python/main.py

```diff
@@ -95,11 +162,24 @@ async def get_image(image_name):
     return FileResponse(image)
 
 
-class Item(BaseModel):
-    name: str
-    category: str
-
+# items.json's pash
+ITEMS_FILE_PATH = Path("items.json")
 
 def insert_item(item: Item):
-    # STEP 4-1: add an implementation to store an item
-    pass
+    # STEP 4-2: add an implementation to store an item
+    # if no json data
+    if not ITEMS_FILE_PATH.exists():
+        with open(ITEMS_FILE_PATH, "w") as f:
+            json.dump({"items": []}, f)
+
+    # open items.json
+    with open(ITEMS_FILE_PATH, "r", encoding="utf-8") as f:
+        data = json.load(f)  
+
+    # add new item
+    new_item = {"name": item.name, "category": item.category, "image_name": item.image_name}
```

Comment:
pydanticの機能を活用すれば一行でかけます〜: https://docs.pydantic.dev/latest/concepts/serialization/#modelmodel_dump
```suggestion
    new_item = item.model_dump()
```